### PR TITLE
HDDS-8851. Recon Heatmap UI - User cannot navigate back from No Data

### PR DESCRIPTION
**Pull request by smitajoshi12**
## What changes were proposed in this pull request?
After Doing Testing in CDPD Cluster Found Below Issues

1) Heatmap refresh was not working once we get no Data acvailable.
2) Space Issue for Smaller Blocks
3) Absolute Path was not working.
4) Empty input was going through api.

## What is the link to the Apache JIRA
https://issues.apache.org/jira/browse/HDDS-8851

## How was this patch tested?
Manually

### CODE REVIEW

**Review by sumitagrawl — APPROVED** [review]
@smitajoshi12 LGTM +1

File: hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/views/heatMap/heatMapConfiguration.tsx [hunk 1]
```diff
@@ -82,8 +82,9 @@ export default class HeatMapConfiguration extends Component {
           tileStrokeWidth: 1,
           colorDomain: [0.000, 0.050, 0.100, 0.150, 0.200, 0.250, 0.300, 0.350, 0.400, 0.450, 0.500, 0.550, 0.600, 0.650, 0.700, 0.750, 0.800, 0.850, 0.900, 0.950, 1.000],
           colorRange: [...colorRange1],
-          groupFill: 'black',
-          nodePadding: 1, //Disatnce between two nodes
+          //groupFill: 'black',
```
Comment (dombizita) [inline hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/views/heatMap/heatMapConfiguration.tsx:85]: is this commented out intentionally? if we can remove it please delete the line

File: hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/views/heatMap/heatMapConfiguration.tsx [hunk 2]
```diff
@@ -82,8 +82,9 @@ export default class HeatMapConfiguration extends Component {
...
+          nodePadding: 1.5, //Disatnce between two nodes
```
Comment (dombizita) [inline hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/views/heatMap/heatMapConfiguration.tsx:86]:
```suggestion
          nodePadding: 1.5, // Distance between two nodes
```

**Comment by dombizita** [comment]
could you please also review @devmadhuu?

File: hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/views/heatMap/heatMapConfiguration.tsx [hunk 3]
```diff
@@ -122,7 +126,7 @@ export default class HeatMapConfiguration extends Component {
     if (params.datum.label !== "") {
         tooltipContent += `<br/>
           File Name:
-          ${params.datum.label ? params.datum.label.split("/").slice(-1) : "no"}
+          ${params.datum.label ? params.datum.label.split("/").slice(-1) : ""}
```
Comment (devmadhuu) [inline hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/views/heatMap/heatMapConfiguration.tsx:128]: What file name is this ? and why removed "no", pls explain.

File: hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/views/heatMap/heatMapConfiguration.tsx (hunk 2 above)
Comment (smitajoshi12) [reply hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/views/heatMap/heatMapConfiguration.tsx]: @dombizita Corrected in latest commit.

File: hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/views/heatMap/heatMapConfiguration.tsx (hunk 3 above)
Comment (smitajoshi12) [reply hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/views/heatMap/heatMapConfiguration.tsx:128]: @devmadhuu 
If lastnode if label was  not present then dont display label in tooltip. "no" just testing string for negative test cases. so corrected in this PR.

**Comment by smitajoshi12** [comment]
> @smitajoshi12 - as discussed, pls show tooltip of access count using "maxAccessCount" for bucket level

Yes COmpleted Change in latest commit.

**Review by devmadhuu — APPROVED** [review]
@smitajoshi12 LGTM +1

**Pull request merged**